projects/hslayers/src/common/layman/access-rights/layman-access-rights.component.ts

```diff
@@ -0,0 +1,26 @@
+import {Component, Input, ViewRef} from '@angular/core';
+import {accessRightsInterface} from '../../../components/add-data/common/access-rights.interface';
+
+@Component({
+  selector: 'hs-layman-access-rights',
+  templateUrl: './layman-access-rights.html',
+})
+export class HsCommonLaymanAccessRightsComponent {
+  @Input() access_rights: accessRightsInterface;
+  constructor() {}
+
+  accessRightChanged(
```

Comment:
We could export this as a function nd then reuse it in other components (draw-layer-metadata.component.ts , add-data-vector.component.ts) (in save-map-advanced-form.component.ts looks similar too, but you know better)

Reply:
Ye sure I forgot about save-map-advanced-form.component.ts. Should be used there as well.  Also forgot to delete  accessRightChanged functions from components other than newly created one... thanks for pointing this out. Was kinda busy and trying to rush everything so I do these unnecessary mistakes